scripts/gha-build

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+
+./validate
```

Comment:
We're already running validate in other workflows, I don't think we need to run it here